Source/Core/InputCommon/InputConfig.cpp

```diff
@@ -33,6 +33,11 @@ bool InputConfig::LoadConfig(bool isGC)
   std::string profile[MAX_BBMOTES];
   std::string path;
 
+#if defined(ANDROID)
+  bool useIRConfig = false;
+  std::string irValues[3];
```

Comment:
Should be named `use_ir_config` and `ir_values`.

Reply:
fixed